### PR TITLE
Feat/create widget

### PR DESCRIPTION
**Pull request by fprebolledo**
## Context 

We are creating a library to embebed a soyio widget.

## What has been doing.

In this PR it is exposed a simple function to create or fill an iframe with the widget. At this moment the widget url is a youtube video, for making tests.

### CODE REVIEW

File: spec/widget.spec.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+// src/sum.test.ts
```
Comment (difernandez) [inline spec/widget.spec.ts:1]: what's this path?

File: spec/widget.spec.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    })
+    it('adds iframe id', () => {
+      const iframe = mountIframeToDOM()
+      expect(iframe?.id).toBe(IFRAME_ID)
+    })
+  })
+
+  describe('when iframe exists', () => {
+    beforeEach(()=> {
+      document.body.innerHTML = '<iframe id="soyio-widget-iframe"></iframe>'
```
Comment (difernandez) [inline spec/widget.spec.ts:28]: You could use the `IFRAME_ID` here

**Review by difernandez — COMMENTED** [review]
some comments!

File: src/constants.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export const IFRAME_ID = 'soyio-widget-iframe';
+export const DEVELOPMENT_WIDGET_URL = 'https://www.youtube.com/embed/0aG9umgGFv0?si=Y7ntdvgcwwxORz4P';
```
[the remark this answers is not in the document]
Comment (fprebolledo) [reply src/constants.ts:2]: I thought leaving it in that way in the meantime to make tests. But I think i will change it even if the real url is not prepared to be embedded.

File: spec/widget.spec.ts (hunk 1 above)
Comment (fprebolledo) [reply spec/widget.spec.ts:1]: Ups. My bad.

File: package.json [hunk 4]
```diff
@@ -25,16 +25,16 @@
   },
   "devDependencies": {
     "@types/node": "^20.10.1",
-    "@typescript-eslint/eslint-plugin": "^6.13.1",
-    "@typescript-eslint/parser": "^6.13.1",
+    "@typescript-eslint/eslint-plugin": "^6.13.2",
+    "@typescript-eslint/parser": "^6.13.2",
     "eslint": "^8.54.0",
     "eslint-plugin-import": "^2.29.0",
     "eslint-plugin-platanus": "^0.1.0",
```
Comment (difernandez) [inline package.json:32]: you could also remove this

**Review by difernandez — APPROVED** [review]
💯

File: package.json (hunk 4 above)
Comment (fprebolledo) [reply package.json:32]: you're right. I will remove it in the next PR. Thanks.

**Pull request merged**